package.json

```diff
@@ -32,10 +36,11 @@
     "prettier:cli": "node packages/build/bin/run-prettier \"**/*.ts\"",
     "prettier:check": "npm run prettier:cli -- -l",
     "prettier:fix": "npm run prettier:cli -- --write",
-    "clean": "lerna run clean --loglevel=silent --parallel",
+    "clean": "rimraf .build && lerna run clean --loglevel=silent --parallel",
```

Comment:
No need to use `rimraf` as an extra dependency. Let's stay with `node packages/build/bin/run-clean`.